alarm/management/commands/price_break_threshold_alarm.py

```diff
@@ -26,9 +31,27 @@ def handle(self, *args, **options):
                 Candle.refresh_candle_data(trade_pair, high_price, low_price)
 
                 if any_of_trade_pair_thresholds_is_broken(trade_pair):
-                    # TODO: make_call()
-                    logger.info('need call')
-                    pass
+                    threshold_brakes = ThresholdBrake.objects.filter(threshold__trade_pair=trade_pair)
```

Comment:
Guys, I need you both to understand how code should be wrapped into functions. The crucial part here is that below is the specific step of logic, which is responsible for calling a user. Why is not it a separate function? Each review I check 2 things: decoupling (how code is divided into functions) and naming. Please make check against them when sending to review.

Reply:
++